utils/version.go

```diff
@@ -43,17 +43,61 @@ const (
 	Less    ComparisonStatus = iota - 1 // -1
 	Equal                               // 0
 	Greater                             // 1
+	InvalidFlags
```

Comment:
This seems out-of-place here. Isn't this an error which should be reported accordingly?

I'd expect the comparison flags to be inlined at every call site. Therefore using an incorrect set of flags is definitely a programmer error and we can panic if the flags are invalid.

Reply:
it's a bad pattern to decide what should do unknown program. To handle panics in golang programs programmer should to wrap this function which will defer function with `recover()`` [call](https://github.com/golang/go/wiki/PanicAndRecover) if he wants to handle an error in place where it has occurred

I added this flag for cases when wasn't used any flag (pass 0) or incorrect value. Function should compare two values and return any less/equal/greater. But in other cases with undefined (oh, maybe use `undefined`?) value used some value that will show it and allow to handle it.

Anyway programmer will test his code with comparisons and catch his mistakes. He can write `if v1.CompareOnly(0, v2) <operator> <value` where operator may be == or != (>, >=, <=, < is unexpected and not supported operators as any of other implementations of `compare` in golang/c/c++). If he expect that result ` == Equal` or ` != Equal`, he will take it anyway if pass incorrect flag : )

or we need +1 opinion how will be better in this case )